LoopKit/InsulinKit/DoseStore.swift

```diff
@@ -758,11 +734,6 @@ extension DoseStore {
                 object.title = event.title
                 object.type = event.type
                 object.mutable = event.isMutable
-                if let type = self.pumpInsulinType {
-                    object.dose = event.dose?.annotateDoseWithInsulinTypeIfNeeded(defaultType: type)
```

Comment:
If we're removing the annotation step, then we will want to include the insulin type info when the dose is created in the pump manager. Guessing you already though of this, but since I didn't find an insulin type branch on the rileylink_ios Github, I wanted to make sure

Reply:
Yes, I do have some changes around this in progress, that aren't pushed yet.